libvmaf/src/feature/cambi.c

```diff
@@ -36,6 +36,9 @@
 /* Visibilty threshold for luminance ΔL < tvi_threshold*L_mean for BT.1886 */
 #define DEFAULT_CAMBI_TVI (0.019)
 
+/* Max log contrast luma levels */
+#define DEFAULT_MAX_LOG_CONTRAST (2)
```

Comment:
Can we add one test case to cover the case when max_log_contrast is not the default?

Reply:
added